<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Marun BKS</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/logos.qrc">
    <normaloff>:/marun_logo/img/marun.png</normaloff>:/marun_logo/img/marun.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(211, 215, 207);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>341</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 215, 207);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res/logos.qrc">:/tf_logo/img/tf.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 14pt &quot;URW Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Belge Kriptolama Sistemi</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>571</width>
      <height>351</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>142</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>50</number>
    </attribute>
    <column>
     <property name="text">
      <string>Kitap / Belge Adı</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ders</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Denetmen</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Teslim Tarihi</string>
     </property>
    </column>
   </widget>
   <widget class="QFrame" name="auth_frame">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>691</width>
      <height>431</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>331</width>
       <height>281</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>271</width>
        <height>232</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">font: 25 19pt &quot;URW Bookman&quot;;</string>
         </property>
         <property name="text">
          <string>Giriş Yapın</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">font: 25 14pt &quot;URW Bookman&quot;;</string>
         </property>
         <property name="text">
          <string>Email / Öğrenci No</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="username_input"/>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">font: 25 14pt &quot;URW Bookman&quot;;</string>
         </property>
         <property name="text">
          <string>Şifre</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="password_input"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="register_button">
           <property name="styleSheet">
            <string notr="true">font: 25 14pt &quot;URW Bookman&quot;;</string>
           </property>
           <property name="text">
            <string>KAYIT OL</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="submit_button">
           <property name="styleSheet">
            <string notr="true">font: 25 14pt &quot;URW Bookman&quot;;</string>
           </property>
           <property name="text">
            <string>ONAYLA</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="forgot_pass_btn">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 25 10pt &quot;URW Bookman&quot;;
color: rgb(0, 0, 127);</string>
         </property>
         <property name="text">
          <string>Şifremi Unuttum</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QFrame" name="register_frame">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>471</width>
       <height>381</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>271</width>
        <height>288</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_8">
         <property name="styleSheet">
          <string notr="true">font: 25 19pt &quot;URW Bookman&quot;;</string>
         </property>
         <property name="text">
          <string>Kayıt Olun</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_9">
         <property name="styleSheet">
          <string notr="true">font: 25 14pt &quot;URW Bookman&quot;;</string>
         </property>
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="reg_username_input"/>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_11">
         <property name="styleSheet">
          <string notr="true">font: 25 14pt &quot;URW Bookman&quot;;</string>
         </property>
         <property name="text">
          <string>Öğrenci No</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="reg_username_input_2"/>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">font: 25 14pt &quot;URW Bookman&quot;;</string>
         </property>
         <property name="text">
          <string>Şifre</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="reg_password_input"/>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_10">
         <property name="styleSheet">
          <string notr="true">font: 25 14pt &quot;URW Bookman&quot;;</string>
         </property>
         <property name="text">
          <string>Kayıt Parolası</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="register_key"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>310</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="reg_cancel_button">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 25 14pt &quot;URW Bookman&quot;;</string>
         </property>
         <property name="text">
          <string>IPTAL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="reg_submit_button">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 25 14pt &quot;URW Bookman&quot;;</string>
         </property>
         <property name="text">
          <string>Kayıt Ol</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>350</y>
        <width>251</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Öğrenci Numaranızı doğru girdiğinizden emin olun!</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>351</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Belge Kodu :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="code_input"/>
     </item>
     <item>
      <widget class="QPushButton" name="add_book_btn">
       <property name="text">
        <string>Kitap/Belge Ekle</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="openbook_btn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>500</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Kitabı Aç</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res/logos.qrc">
      <normaloff>:/buttons/img/repeat.png</normaloff>:/buttons/img/repeat.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>tableWidget</zorder>
   <zorder>openbook_btn</zorder>
   <zorder>refresh_btn</zorder>
   <zorder>auth_frame</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>745</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res/logos.qrc"/>
 </resources>
 <connections/>
</ui>
